spec/resty/concurrent/executor_spec.lua

```diff
@@ -0,0 +1,15 @@
+local Executor = require('resty.concurrent.executor')
+
+describe('ImmediateExecutor', function()
+    describe('.from_options', function()
+        it('has default executor', function()
+            assert(Executor.from_options{ })
```

Comment:
Shouldn't we check that the default is the immediate one?

Reply:
Yep, why not.